### DISCUSSION TITLE
Diamonds value source

### DISCUSSION THREAD

**Discussion by 0xNordian** [deso-protocol/dips#26] [Medium CIPs]
name: Diamonds value source
about: Change the source of the diamond's value
title: N/A
labels: Diamonds, Content Reward
assignees: N/A

**One-Line Description**
-------------------------
Diamonds 💎 to be a $clout transfer from the sender balance as opposed to the sender creator coin.

**Explanation and Motivation**
-------------------------------
I understand diamonds as a special recognition for a piece of content a creator has created. The sender wants to reward that content by giving diamonds. For the diamonds to become a quick and frictionless source of income, I think the transfer should be in $clout (directly to the 'wallet balance' and avoiding the act of selling) and not by transferring creator coin (we also have the Send Creator Coins function as a feature)

**Some benefits:**
-------------------
1) Diamonds can be sold without impacting the original creator coin price (If $CLOUT goes to the wallet balance this step is no needed)
2) The perception as a creator that you are having an income for your content increase
3) Sender 'creator coin' volatility will not impact the value of the diamond
4) Avoiding community 'drama' for the rewarded creator for selling diamonds to have liquidity

**Comment by tijno** [comment]
Thanks for suggesting this. Its been a frequent debate on BitClout for sure.

Im neither for nor against right now and interested in how the discussion evolves.

To balance it out, here are benefits from having diamonds in creator coin:

* More personal: i am tipping you my coin because i love your content. Payment in clout feels less personal and more "transactional" between customer and supplier.

* Additional upside: a level 4-6 diamond from a new account gives considerable upside in the long run.

* Ownership dilution: people are less likely to buy 0.01 of my coin, especially when price is higher. Diamonds allows me to over time increase the number of accounts that own some of my coin which is a good thing.

Of course there is the challenge of diamond dust - but solutions are starting to be build for this (eg cloutcli).

**Reply by 0xNordian** [reply]
Thanks for the feedback. My comments:

> More personal: i am tipping you my coin because i love your content. Payment in clout feels less personal and more "transactional" between customer and supplier._

I think this is easily solvable by not taking out the capacity to recognize who has given you the diamonds (as it is now); the only thing it's changing is the source of value.

> Additional upside: a level 4-6 diamond from a new account gives considerable upside in the long run._

Agree. However, I personally see diamonds as a short term liquidity option for the creator that has been rewarded for a piece of content. But I totally see your point. 
(As a disclaimer, I'm not proposing to change anything of the current mechanics of the diamonds systems, just the value source)

> Ownership dilution: people are less likely to buy 0.01 of my coin, especially when price is higher. Diamonds allows me to over time increase the number of accounts that own some of my coin which is a good thing.

This is an excellent point. If the current system remains, the only thing I would like to suggest is the capacity to sell diamonds in a bulk way or else is operationally inefficient even in this early stage,

**Reply by tijno** [reply]
> not taking out the capacity to recognize who has given you the diamonds

this remains in place for sure.

but my point was more about me sharing my coin feels more personal. It wanst about the notification / diamonds received list. 

> short term liquidity

the amounts in general are so small and selling is hard - is it really being used for liquidity?

maybe im exception but ive not sold any of received diamonds.

also - upside from cc will be huge - consider DAU 1000x of today - eg twitter level. Todays cc received through diamonds will i think all show massive upside

**Reply by HPaulson** [reply]
@tijno 

> the amounts in general are so small and selling is hard - is it really being used for liquidity?
> maybe im exception but ive not sold any of received diamonds.

I think this is the entire point. Diamonds COULD be used to be a valid and legit source of income for creators, but right now they just aren't viable and are just a way to have someone in your wallet. It's more like an appreciation token than actually supporting the creator.

**Reply by 0xNordian** [reply]
> but my point was more about me sharing my coin feels more personal. It wanst about the notification/diamonds received list.

It's a valid point to take into consideration.

> the amounts in general are so small and selling is hard - is it really being used for liquidity?

**If they go directly to your wallet, it would be instant liquidity.** Furthermore, by not selling diamonds, a creator can have the diamonds as a placeholder in their profile to show who has supported them with the clout tally beside the sender profile (if I not mistaken you lose this reference as soon as you sell the diamonds). 

The liquidity aspect incentives content creation and avoid the problem of selling diamond mechanics.

**Reply by tijno** [reply]
re @hpaulson 

> Diamonds COULD be used to be a valid and legit source of income

i think they already are. Its just hard to liquidate. Thats where the friction is i think. If that was easier - it would solve the issue no? 

re @N0rdian

> a creator can have the diamonds as a placeholder

im not aware of that going away when cc is sold. 


In addition -

creators will have nfts as an option of earning a very good income from their creations.

NFTs are paid in clout. Ongoing for live as they are bought and sold.

Diamonds are different - they are like tipping. 

Im still not convinced and of course dont need to be. Healthy discussion - good.

Hopefully some others will contribute.

**Reply by HPaulson** [reply]
@tijno See my full comment below. This would solve one issue, but not the major two problems that diamonds in cc have.

**Comment by HPaulson** [comment]
I couldn't upvote this suggestion enough. Having diamonds be in Clout makes them a legitimate source of support. More info:

By having diamonds be in the creator coin, the user is more hesitant to give such diamonds, as it's harmful. When giving you a diamond, which is a form of monetary support, you must sell that diamond in order to support yourself (Which, note, is a huge hassle in itself). When one sells such, you: A. Hurt the investments in that supporter's own coin, and B. Hurt the investors of the individual who supported you -- making the monetary loss of the supporter much greater than just the amount actually given in the diamond's value.

Having the diamonds be in clout eliminates both A and B above, as well as give creators a viable way of gaining support from users. Diamonds, at scale and by using clout directly, could be a legitimate profit source for creators -- allowing them to monetize EVERY post, without a brand deal (Where as traditionally creators on social media must have a deal to monetize a post).

**Reply by malaybanerjee1** [reply]
I agree with you. Diamonds in Creators own coin making loss of Investors. Diamonds in clout mean everyone has to buy certain amount of $Clouts . It's will increase the price of $clout. All Creators benefitted by getting clout.
@malay11

**Comment by gustavoflanagan** [comment]
Separate but related:

If $clout appreciates relative to fiat, diamonds will be more far valuable. There will be far less diamonds given if the lowest is $1 USD. Presumably, the $0.01 diamond is the desired baseline.

Maybe, and I'm sorry for this, we should peg it to USD but pay it in $clout?

**Reply by HPaulson** [reply]
I see 0 reason why we shouldn't do this, honestly

**Comment by dgsus** [comment]
i'm more incline to agree with narrative that diamonds should not be seen a primary source of short term liquidity, but as a long term investment from both sides. 

having said that, im in favor of each user deciding on how they wish to handle their received diamonds.
i'm in favor of an easy native way (frontend src) to sell diamonds in bulk as the user receiving them wishes - as @N0rdian mentioned. 

perhaps options should be given to sell diamonds, something like a x number of diamonds from all diamonds received to make the implementation easier, for now, but efficient. depending on the diamonds received, selling such a low amount as 1 diamond wouldnt make any impact on the creator that gave that diamond, but could make a huge impact on the receiver wallet ;)

e.g a "global" sell option at the top of the creator diamonds area in which a user can input sell 1 diamond 0.01$ (from all received diamonds)

sell 2 diamonds, etc
even all diamonds

i know tools like cloutcli are already addressing this which is really cool, but its not for the average user.

**Reply by tijno** [reply]
> having said that, im in favor of each user deciding on how they wish to handle their received diamonds.

receiver setting maybe ? 

receive diamonds in:
[ ] $clout
[ ] creator coin

[suggested by sidz here](https://bitclout.com/posts/929df2ae232294d25b898cd89d10c070c4805b28a0e6a45e42f95c1c82d6efaf?feedTab=Following)

**Reply by dgsus** [reply]
cool, transaction would take care of burning the coin and sending the clout 
options on both sides, which i forgot to mention above.

cheers!

**Reply by HPaulson** [reply]
@Tijno This is a great idea. The only problem is: A user may not want to give clout, only their cc, and not know what the receiver has set. Maybe first time you diamond a user it'll tell you? Or a tiny bit of text at bottom of post?

**Reply by dgsus** [reply]
@HPaulson  "...The only problem is: A user may not want to give clout, only their cc, and not know what the receiver has set..."

why not have the protocol fulfilling your wish as mentioned above? e.g i send diamonds -> code checks what is receiver setting, sells cc and sends clouts if receiver setting is to receive in clout -> if not send cc as usual.

**Reply by HPaulson** [reply]
This doesn't solve the problem

**Reply by AikonMishima** [reply]
Possible solution:
On diamond screen, put a checkbox:
☑️Give in Creator Coin
☑️Give in $CLOUT

this would allow a choice and solve the problem.

standard selection can be made in selection settings, standard is « give in creator coin »

Great idea BTW @N0rdian

**Reply by dgsus** [reply]
> This doesn't solve the problem

Ah, indeed, I re-read it and you are right, but if the receiver wants it converted into clout they could anyway, even thought is a manual activity now.

I would argue that if the sender would want that the receiving side lists the sender coin in the receiver wallet, he is already forcing this upon the receiver. Which, IMO, I don't believe we should "enforce".

**Reply by arhebbar** [reply]
I agree with @HPaulson. This is better off as a sender setting in one of two ways and the sender should be able to choose that (not the receiver)

As a sender of diamonds, these should be my options
1. If I am particular about maintaining my own Creator Coin Price and want to reward a small Creator in $CLOUT, I should be able to do that (e.g. for the Gempay usecase)
2. If I see the creator as an investor or if he's already my investor and I want him to deepen the relationship, I should be able to give the diamonds as Creator Coin.

From the Creator Side, not showing the really small CCs (say < 0.1$) and more importantly, selling all Creator Coins below a certain # in a bulk sweep, will allow users to keep their wallets clean without much hassle.

**Reply by blackstock** [reply]
Okay, this is turning into an epic bikeshedding session.

I'd argue to keep this simple.

Limited time and resources and I'd wager the average creator doesn't want random peoples coins, so why waste time building that? There are countless more important features to build. There is already a mechanism to send creator coins to other users, so that can be utilized on the occasion a user wants to send another user their coins. Tipping in CLOUT is simple and intuitive.

**Reply by arhebbar** [reply]
If I have to make a choice, I'd go with $CLOUT any day. I have spent half a day cleaning up by 0.00$ coins this weekend and I can only imagine how bad it will be when we have millions of users here (some day). Also, if the change to $CLOUT is not done. A sweep to $CLOUT automatically or with one click should move up in priority.

**Comment by blackstock** [comment]
This feels like a no-brainer to me. Content creators who want to monetize using this tool want money, not someone's shitcoin (no offense to anyone - my coin is a shitcoin too).

If content creators can not easily and quickly monetize using this tool the value proposition isn't there. It's already an inferior tool to Twitter in terms of search, reach, etc and adding friction by forcing them to sell random coins they're being gifted is a horrible UX (and certainly worse than being tipped USD on Twitter).

**Reply by gustavoflanagan** [reply]
As BitClout scales not every user will be a "creator" and hold a creator coin. In fact, this may be the vast majority of users. Presumably these users will want to diamond and may be prolific diamonders but never post one post or video. So I wont refer to them pejoratively but I will say their creator coin is not going to be valuable. 

Pay it in clout but peg it to a USD amount .

**Comment by Shubs21** [comment]
Majority of people keep most of their $clout in Creator Coins and not in Bitclout. 

Having 💎 payouts in $clout would just decrease the number of 💎 Transactions.

As far as the hassle involved in selling 💎s is considered, we just need a 'sell coins under $x' feature.

Moreover, when a change like this is being made, people should think about the small creators that constitute 70% of the platform.

1. The 💎s sent by small market cap creators are much more valuable.

2. They have a lot less $clout in hand, having them 💎 in $clout is 📉

The supporter should have an option either way so I guess @tijno's idea of selecting mode of 💎 pay-outs is awesome! It has a ton of other benefits apart from just helping Creators monetise easily.

**Reply by tijno** [reply]
not my idea :)

sidz

https://bitclout.com/posts/929df2ae232294d25b898cd89d10c070c4805b28a0e6a45e42f95c1c82d6efaf

agree with all you said though

**Reply by 0xNordian** [reply]
> Majority of people keep most of their $clout in Creator Coins and not in Bitclout. Having 💎 payouts in $clout would just decrease the number of 💎 Transactions.

This is a good point and an interesting theory. 
Since given and received diamonds is public information in the front end, and I think they impact your 'brand reputation' in the platform, I wonder if the proposed change will be applied, would this modify the behaviour you are describing? (having the majority of the $clout in 'CC).

Thanks for your feedback!

**Reply by Shubs21** [reply]
> > Majority of people keep most of their $clout in Creator Coins and not in Bitclout. Having 💎 payouts in $clout would just decrease the number of 💎 Transactions.
> 
> This is a good point and an interesting theory.
> Since given and received diamonds is public information in the front end, and I think they impact your 'brand reputation' in the platform, I wonder if the proposed change will be applied, would this modify the behaviour you are describing? (having the majority of the $clout in 'CC).
> 
> Thanks for your feedback!

I don't think it will modify this behaviour. The reason people allocate most of their funds to CCs is to maximise their investment. Increase in Creator Coins can easily beat the increase in Bitclout Price. 

Since this has close to 0 relation with Diamonds, everything will remain the same.

**Comment by ShhubhamA** [comment]
I agree with @N0rdian  There is a lot of community 'drama' when someone sells the diamonds !

 Also one added benefit  i see is it would prevent  people misusing the diamonds feature in creator coin by giving his/her friends  a lot of diamonds and then asking him  to sell/rug

**Reply by marnimelrose** [reply]
This truly would be a way to appreciate people's content monetarily. For instance, there is an issue that I highlighted on Twitter a couple of days ago, "Black creators are striking on TikTok because they are not being compensated for their work. " - here's a link to the post https://twitter.com/marnimelrose/status/1413348035833200643 
If we had diamonds working as clout it could be a way to bring others over from different platforms. As others have said here, **_people don't care about your coin they care about your compensation._**

**Comment by bloated-devish** [comment]
In practice, it appears the selling "drama" only happens when the coin is worth a good amount. Even if 100 people give a single diamond, its too much work to "sell it" to cash out CLOUT. 

## Idea 1 
May be at 5+ diamonds, there should be an option to receive clout instead of diamonds. 

##  Idea 2
An alternative idea would be for creators to leave a tab on their account, so if their coins are sold, up to the wallet earmarked for it, they'll just be bought back by the creator, so the coin value stays the same. If they don't have wallet balance, the sale will simply run through affecting the coin value.

**Comment by sull** [comment]
In the early days when I was thinking about tipping, crowdfunding and payments ([see this post on bitclout](https://bitclout.com/posts/8f26f629388fa8c56692267596494411f43c05e611f2786d990af2f525343a2c)) I wanted to see less creator coin activity and more direct sending of $clout. I also saw value in users being seen and listed as creator coin holders/stakers. 

I feel like this conversation has some parallels. 

If sending diamonds as $clout could also add you (the sender) to the recipient's "patron list" and also the sender could receive a "badge" (representing value tiers) seen in a list on their own profile next to each account name that recieved funds from you, it could be interesting and deal with some of the nuances of what is being discussed here.

I tend to look at creator coins as a seperate thing when it comes to simple monetary funding actions and prefer that the coins don't get intermixed by default. We have "send creator coin" capability as stand-alone so why complicate tipping etc buttons/actions?

**Comment by tijno** [comment]
I have a feeling it may be hard to reach consensus on this proposal at either end of the opinions:
 "change to pay diamonds in cc" vs "leave as is".

Any views @N0rdian on what the next step is?
 
Right now seems there are 2 option:
1. You have enough buy-in from some of the community clearly - so if core approve this could go ahead. 
2. Implement the "let the user decide" option I [posted above](#discussioncomment-986824).

Not a great fan of either 1 or 2 - but interested to see if you have any other ideas?

Also aware its only been 3 days since posting. Probably should give it at least a week :)

Hopefully some input from core team will help to move this forward too.

**Reply by bloated-devish** [reply]
A third option is let the creator leave a tab - so that if the diamonds are sold, the chain can make the creator automatically purchase them back. This so that diamond holders aren't put in a position today where its awkward to sell - and for the creator this isn't a big deal - since the amount that they'll probably have to leave in their wallet earmarked for potential diamond sale by supporters is usually not high - this is also an opportunity for the creator to buy their own coins back (and this should be encouraged - if possible).

Does this idea sound reasonable?

**Reply by tijno** [reply]
Im not sure i understand - could you walk me through an example?

**Reply by bloated-devish** [reply]
# Case 1 (Happy path)
1. Supporter tijno has 100$ in their wallet (another variation is a diamond sub-wallet that has 100$ in it).
2. tijno drops a 5 diamond to Creator bloated for their amazing CIPs 🤪. Diamonds are worth roughly 87$.
3. bloated chooses to sell tijno's diamond value in creator coin. On the seller page, bloated notices that tijno will pick up the tab if bloated sells it. Feels relieved selling it.
4. The chain notices that tijno has earmarked 100$ in the diamond sub-wallet and deducts 87$, leaving 13$ in the diamond sub-wallet. 
5. tijno's coins don't change in value (even by a small value). bloated feels good cashing their diamonds and not worried about hurting tijno coin price.
6. tijno happy since their coin didn't drop in price.

# Case 2 (Impaired path)
1. Supporter tijno has 55$ in their wallet (or 55$ in their diamond subwallet).
2. same as case 1
3. same as case 1
4. Chain notices tijno has 55$, and buys 55$ worth of tijno cc. The remaining 32$ worth of coins sold by bloated drops tijno's coin price by a small value.

# Case 3 (Today)
1. Supporter tijno has 100$ in their wallet (another variation is a diamond sub-wallet that has 100$ in it).
2. tijno drops a 5 diamond to bloated. It is roughly 87$.
3. bloated chooses to sell tijno's diamond value in creator coin. Dilemma to sell - tijno is my biggest supporter. If I sell, will tijno feel bad that I sold their coin? 

\
Thanks\
Bloated

PS: If the creator coin buys dip into main wallet instead of diamond sub-wallet, there needs to be a setting to "enable it".

**Reply by tijno** [reply]
Thanks for the walkthrough - much clearer.

However i think this way overcomplicated things and would put additional pressure on protocol and user education. 

Im not sure all this is warranted for an issue that is only affecting/seen by some users.

**Reply by 0xNordian** [reply]
> I have a feeling it may be hard to reach consensus on this proposal at either end of the opinions:
> "change to pay diamonds in cc" vs "leave as is".
> 
> Any views @N0rdian on what the next step is?

Apologies for the delay. 
It's hard for me to say since it is just a proposal from my side, and I'm not trying to convince anyone. I don't know if there is enough quorum to determine what the community thinks about this matter. Maybe the only following proposal is to create a poll and share it in the bitclout.com node for a couple of days and see what feedback we get. I would like to know the opinion of this group to proceed if we agree with that mechanism.

> Right now seems there are 2 option:
> 
> 1. You have enough buy-in from some of the community clearly - so if core approve this could go ahead.
> 2. Implement the "let the user decide" option I [posted above](#discussioncomment-986824).
> 
> Not a great fan of either 1 or 2 - but interested to see if you have any other ideas?
> 
> Also aware its only been 3 days since posting. Probably should give it at least a week :)
> 
> Hopefully some input from core team will help to move this forward too.

Agree. I think it would be great to count on the opinion of @diamondhands0 on this since the suggested change is related to the ethos and vision of the platform.

**Reply by tijno** [reply]
Agree about hearing the teams opinion on this for sure

**Reply by Elrickerikose** [reply]
Actually, new thing needs to be introduced, emeralds or something, because diamonds should be in CLOUT , but if someone likes another creator or/and investor so much, that wants this creator or / and investor to hold his coin, that should be emerald 😊

**Comment by omicreativedev** [comment]
We added Diamonds later. Why not just add a secondary item such as a 🚀 rocket (call it boost) that tips in CLOUT at
.01, .10, 1., 10., 100 with the same type of modal as the diamond one. And the user would need clout in wallet to offer it. It could also have an ARE YOU SURE popup (which I think should be on Diamonds also)

This would lead way for new innovations.

Aside from all the above issues mentioned, diamonds are also a way people can change accounts and boost themselves up on leaderboard, and sell out of their coin without notifying others. 

I think we should get rid of the daily diamond leaderboard. It fine for dapps, but since it's so rigged it's a bad look on main.
The highest rising creators, projects, top creators of all time (minus unclaimed accounts) are more aligned with the concept of trading anyway.

Alternatively, maybe have it so a user can choose to autosell diamonds. And then when people diamond have some kind of icon that alerts that this user chooses to autosell diamonds under XYZ value, then that means we only accept diamonds as coin when over .1 for instance. 

I think removing the FAME factor for giving and receiving diamonds on a leaderboard would reduce people using moving diamonds for notoriety. Right now I know 2 accounts that just diamond each other to stay in the toplist.

**Reply by danwils33** [reply]
This is brilliant. Seems like it would accomplish @tijno suggestion of letting the community pick how they want to reward/be rewarded but in a way that removes a layer of complexity. Granted, some will still receive cc's who would rather have $clout, but the sell option will remain.

**Reply by omicreativedev** [reply]
I agree. I think Diamonds will still remain super attractive! They're fricken DIAMONDS ya know?! Plus, IOS is being a sucker about Diamonds. So even if we change it to clout, the 3 major ios apps won't be able to use it smh.

 Screw them. Give them rockets to contend with next while they figure out what to do with it. But yeah, it does seem to semi-resolve about 3 different unrelated issues here while avoiding several major new issues.

**Comment by omicreativedev** [comment]
Also want to add that many dapps that use diamonds might be mad that we changed their business model. Such as gempay and others. Dapp developers don't get compensated when they use their time to make something and suddenly a change in the system breaks or disables their dapp. So I wouldn't change diamonds now that it is exists- even though I voted for it earlier in the poll I have second thoughts.  I would suggest instead of doing a lot of speedbump work around the idea. and the other ideas of autoconverting amounts and such. removing leaderboards. Make a modal that alerts coinholders when a user diamonds more than 10% of their coin at a shot, etc.

**Comment by omicreativedev** [comment]
Added more on the concept of Boost Pay in another CIP using QR codes. Note, I am not talented enough at Go or Angular to take on such an effort or alone. But I think this would actually work given we iron out any kinks.

**Comment by 0xNordian** [comment]
Hi, good morning everyone!
Here you have the final results of the survey I performed:

<img width="650" alt="Screenshot 2021-07-19 at 09 22 14" src="https://user-images.githubusercontent.com/85548493/126119383-2e180d1f-8765-4ebe-857b-607f21b3afca.png">

- If someone wants to check the spreadsheet, here you have the [link](https://docs.google.com/spreadsheets/d/1mv8hd268rkYl44pfVonKr3ggOZtyv--uQXmhiLrmugs/edit?usp=sharing).
- If someone would like to access the document version history for audit purposes, please let me know to give you access.

I did my best to get as many responses as possible, and I thank those who recloted the survey. However, I'm not satisfied with the volume of responses since I would like more people in the community to participate in making a decision. On the other hand, and as mentioned in the past would be great to count on the input of someone from the core team around this matter (@diamondhands0, @maebeam, @lazynina, @bluepartyhat, @redpartyhat)

PS: If we change the vote of @teambrixmusic to ('in Creator Coin') as per her [comment](https://github.com/bitclout/cips/discussions/26#discussioncomment-1012592), the final result would be:

<img width="412" alt="Screenshot 2021-07-19 at 09 33 32" src="https://user-images.githubusercontent.com/85548493/126120744-e3c3aa66-6df2-402a-aa60-975e92b397a0.png">

Best regards,

**Reply by tijno** [reply]
I thought the idea was to get wider community input into this from the survey? This is why i did not vote specifically.

What does the result look like if we remove the devs involved here from the survey?

And does the lack of community engagement mean they dont care or didnt see it? Probably the latter.

**Reply by 0xNordian** [reply]
> I thought the idea was to get wider community input into this from the survey? This is why I did not vote specifically.

I think everyone should participate in the survey to facilitate the counting process. I cannot see who upvoted here in GitHub to remove them from the survey (It is my first time here, so maybe there is a way that I'm not aware of). In any case, the form is still open.

> What does the result look like if we remove the devs involved here from the survey?

Who should I count: comments, upvotes, participants? As mentioned before, I think the easiest way would be to all participate in the same survey.

> And does the lack of community engagement mean they dont care or didnt see it? Probably the latter.

I cannot answer if they do care or not since I don't have access to a representative data sample; I will be inclined to think that was a problem of reach.

**Comment by redpartyhat** [comment]
All-- thanks for the great discussion on this!  After reading through the conversation and discussing with the core team thoroughly, there is strong support for diamonds paid in CLOUT.  I will create a CIP.

**Reply by tijno** [reply]
im not sure there was strong support for this change, and several alternative approaches were outlined that would address the problems mentioned. 

i for one will be disappointed by the move away from diamonds being paid in creator coin

but i get that there may be other benefits as a result of this move that are at play here as well.

eg - easier to demonstrate to new users what the benefits are of diamonds, and reducing large number of small value transactions on chain as a result of converting diamonds to clout.

**Reply by ItsAditya-xyz** [reply]
+1 to what tijn said! 
if diamonds will be paid in CLOUT then what is the use case of own creator coin  ?

**Reply by blackstock** [reply]
> +1 to what tijn said! 
> 
> if diamonds will be paid in CLOUT then what is the use case of own creator coin  ?

Not tipping, IMO.

Creator coins can be used to determine dividends for holders, to earn royalties, and you can use creator coins to determine tiers of exclusive content/access.

Would you ever tip someone on Twitter with shares of Tesla? That's essentially what we're talking about here. There is a simple mechanism to send creator coins to people on the platform already if you want to give out your own coin.

Celebrities/influencers will want money NOT 100,000 random shit coins (my coins is included in the shit coin list).

That's my opinion at least...

**Reply by arhebbar** [reply]
After reading thru this, @teambrixmusic idea on an alternative icon (rocket or even $) to pay in $CLOUT would be a better option compared to complicating what a diamond means. Like I outlined in one of my comments above, giving the "Diamond Giver" the option to reward the creator in $CLOUT or his own Creator Coin as suggested by @tijno first in this thread is the best option.

**Reply by ShhubhamA** [reply]
I don’t like the different type of tipping options. Tipping just in clout would be much better.
I hope we don’t over complicate things  by focusing much on financial aspect and  don’t forget the social aspect :)
Just make sure this is as easy to use for a new onboard so that in future we don’t need to start a course to educate people on how to use the platform :)

**Comment by tijno** [comment]
It would be good to see some of the people that discussed this change here review / comment on the actual CIP which is in progress?

#109

eg @N0rdian @teambrixmusic @bloated-devish @HPaulson (leaving out dgsus as he's on holiday for 3 weeks)

please take time to review the PR @redpartyhat submitted if you haven't already. Its short and concise so will take maybe 10 min of your time.

After all this can go ahead as long as RPH and DH are happy with it - so your input / suggestions / support is important now.

> The community and core devs will review your CIP PR and provide feedback. Once at least two core devs approve, your PR can be merged and is now an official CIP.
